.github/workflows/linting.yaml

```diff
@@ -9,16 +9,16 @@ jobs:
       - uses: actions/checkout@v4
 
       - uses: astral-sh/ruff-action@v3
-        
+
   ruff-format-check:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
 
       - uses: astral-sh/ruff-action@v3
         with:
-          args: format --check --diff
-        
+          args: format --check --diff --fix
```

Comment:
this is a CI action, not to automate formatting the code. so the `--fix` shouldn't be here

Reply:
another point on this - if I push a commit, then an action adds an extra commit remotely, if I do not pull the changes down then try to push some extra commits I'm going to get rejected by github. this could be confusing if not expecting it